dev/how-to/deploy/data-migration-with-binary.md

```diff
@@ -244,124 +242,124 @@ Now, a DM cluster is successfully deployed.
-name: test
-task-mode: all
-is-sharding: true
-meta-schema: "dm_meta"
-remove-meta: false
-enable-heartbeat: true
-timezone: "Asia/Shanghai"
-
-target-database:
-  host: "192.168.0.3"
-  port: 4000
-  user: "root"
-  password: "" # if the password is not empty, you also need to configure the encrypted password using dmctl
-
-mysql-instances:
-  - source-id: "mysql-replica-01"
-    black-white-list:  "instance"
-    route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
-    mydumper-config-name: "global"
-    loader-config-name: "global"
-    syncer-config-name: "global"
-
-  - source-id: "mysql-replica-02"
-    black-white-list:  "instance"
-    route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
-    mydumper-config-name: "global"
-    loader-config-name: "global"
-    syncer-config-name: "global"
-
-black-white-list:
-  instance:
-    do-dbs: ["~^sharding[\\d]+"]
-    do-tables:
-    -  db-name: "~^sharding[\\d]+"
-       tbl-name: "~^t[\\d]+"
-
-routes:
-  sharding-route-rules-table:
-    schema-pattern: sharding*
-    table-pattern: t*
-    target-schema: db_target
-    target-table: t_target
-
-  sharding-route-rules-schema:
-    schema-pattern: sharding*
-    target-schema: db_target
-
-mydumpers:
-  global:
-    mydumper-path: "./bin/mydumper"
-    threads: 4
-    chunk-filesize: 64
-    skip-tz-utc: true
-    extra-args: "--regex '^sharding.*'"
-
-loaders:
-  global:
-    pool-size: 16
-    dir: "./dumped_data"
-
-syncers:
-  global:
-    worker-count: 16
-    batch: 100
-
-```
+    {{< copyable "" >}}
+
+    ```yaml
+    ---
+    name: test
+    task-mode: all
+    is-sharding: true
+    meta-schema: "dm_meta"
+    remove-meta: false
+    enable-heartbeat: true
+    timezone: "Asia/Shanghai"
+
+    target-database:
+    host: "192.168.0.3"
+    port: 4000
+    user: "root"
+    password: "" # if the password is not empty, you also need to configure the encrypted password using dmctl
+
+    mysql-instances:
+    - source-id: "mysql-replica-01"
+        black-white-list:  "instance"
+        route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
+        mydumper-config-name: "global"
+        loader-config-name: "global"
+        syncer-config-name: "global"
+
+    - source-id: "mysql-replica-02"
+        black-white-list:  "instance"
+        route-rules: ["sharding-route-rules-table", "sharding-route-rules-schema"]
+        mydumper-config-name: "global"
+        loader-config-name: "global"
+        syncer-config-name: "global"
+
+    black-white-list:
+    instance:
+        do-dbs: ["~^sharding[\\d]+"]
+        do-tables:
+        -  db-name: "~^sharding[\\d]+"
+        tbl-name: "~^t[\\d]+"
+
+    routes:
+    sharding-route-rules-table:
+        schema-pattern: sharding*
+        table-pattern: t*
+        target-schema: db_target
+        target-table: t_target
+
+    sharding-route-rules-schema:
+        schema-pattern: sharding*
+        target-schema: db_target
+
```

Comment:
The relative indentation in this YAML file is wrong. Please check the original indentation and fix it.